src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/README.md

```diff
@@ -30,8 +30,7 @@ Telemetry correlation http module enables cross tier telemetry tracking.
 
         public void OnNext(DiagnosticListener listener)
         {
-            if (listener.Name == "Microsoft.AspNet.TelemetryCorrelation" ||
-                listener.Name == "System.Net.Http" )
```

Comment:
Was this used previously?

Reply:
I don't think so. My guess is the example snippet came from some code listening to this module and something else instrumenting outgoing requests?

Reply:
Roger that, thanks!